src/constants/nodes.ts

```diff
@@ -32,6 +33,7 @@ export enum MiscNodeType {
     ChongqingRTTextLineBadge = 'chongqingrt-text-line-badge',
     ShenzhenMetroNumLineBadge = 'shenzhen-metro-num-line-badge',
     MRTDestinationNumbers = 'mrt-num-line-badge',
+    MRTLineBadge = 'mrt-line-badge',
```

Comment:
Let's append it to the end so that it follows the sequence of addition dates.

Reply:
It seems we forgot the line badge when we added most of the Singapore MRT style. 

I think it should be placed just behind the "MRT destination numbers", as they are in the same city.